CMSgov/onemac
# oy2-22312 - update TE type label

Story: https://qmacbis.atlassian.net/browse/OY2-22312
Endpoint: See github-actions bot comment

### Details

Update TE label on package list

### Changes

- package list

### Test Plan

1. Login as statesubmitter
2. Navigate to package details
3. Verify existing temporary extension type
4. submit new TE for b or c
5. Verify extension type on package list
